.github/workflows/e2e.yml

```diff
@@ -26,10 +26,14 @@ jobs:
       - name: Reset NX
         run: npx nx reset
 
+      - name: Playwright Last Failed action
+        id: last-failed-action
+        uses: currents-dev/playwright-last-failed@v1
+        with:
+          pw-output-dir: playwright-report
```

Comment:
This is the deepest common shared parent output directory of the projects being run (set in the project.json files)